amethyst_network/src/utils.rs

```diff
@@ -0,0 +1,37 @@
+extern crate bincode;
```

Comment:
That's from all the way when I started rust. The extern crate should go into lib.rs